datastore/gcloud/aio/datastore/value.py

```diff
@@ -38,10 +39,10 @@ def from_repr(cls, data: Dict[str, Any]) -> 'Value':
                 elif value_type == datetime:
                     value = datetime.strptime(data[json_key],
                                               '%Y-%m-%dT%H:%M:%S.%f000Z')
-                elif value_type == cls.key_kind:
-                    value = cls.key_kind.from_repr(data[json_key])
-                elif value_type == LatLng:
-                    value = LatLng.from_repr(data[json_key])
+                elif value_type in [cls.key_kind,
```

Comment:
Please use a set here rather than a list to match style with the rest of this project.